codelists/models.py

```diff
@@ -157,6 +157,9 @@ def latest_visible_version(self, user):
 
         return self.visible_versions(user).first()
 
+    def has_visible_versions(self):
+        return True if self.versions.filter(status="published") else False
```

Comment:
This would be more idiomatic:

```
return self.versions.filter(status="published").exists()
```

Alternatively, if `.exists()` didn't, um, exist, then you could do

```
return bool(self.versions.filter(status="published"))
```

which I think is more idiomatic than the ternary conditional.

Reply:
Oh, of course that's much neater! 

I've amended the commit with this change and the renaming of functions suggested below.